hazelcast/src/main/java/com/hazelcast/nio/ascii/TextReadHandler.java

```diff
@@ -62,7 +62,7 @@
 
     private static final Map<String, CommandParser> MAP_COMMAND_PARSERS = new HashMap<String, CommandParser>();
 
-    private static final int CAPACITY = 500;
+    private static final int CAPACITY = 5000;
```

Comment:
not sure about having a static capacity. a dynamically growing buffer might be better, I think.